packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/OVADetailsSection.tsx

```diff
@@ -93,9 +94,16 @@ export const OVADetailsSection: React.FC<DetailsSectionProps> = ({ data }) => {
         }
         helpContent={
           <Text>
-            {t(
-              'NFS mount end point should be in the form NFS_SERVER:EXPORTED_DIRECTORY, for example: 10.10.0.10:/ova',
-            )}
+            <Trans t={t} ns="plugin__forklift-console-plugin">
+              URL of the NFS file share that serves the OVA.{'<br><br>'}
+              Ensure the URL is in the following format: {'<strong>'}nfs_server:/nfs_path
+              {'</strong>'}, where:{'<br>'}
+              {'<strong>'}nfs_server:{'</strong>'} An IP or hostname of the server where the share
```

Comment:
Maybק "the"?

Reply:
@RichardHoch Do you mean replacing "An IP or hostname of.."
with: "The IP or hostname of..." ?

Reply:
@sgratch Yes, that is my suggestion if in Hebrew you would say "ה" If not, then it's fine as is.